sqlite-android/proguard-rules.pro

```diff
@@ -4,6 +4,7 @@
 }
 -keepnames class io.requery.android.database.** { *; }
 -keep public class io.requery.android.database.sqlite.SQLiteFunction { *; }
+-keep public class io.requery.android.database.sqlite.SQLiteConnection { *; }
```

Comment:
The following also seem to work:

```
-keepclassmembers public class io.requery.android.database.sqlite.SQLiteConnection { *; }
-keepclasseswithmembernames public class io.requery.android.database.sqlite.SQLiteConnection {
    native <methods>;
}
```

I'm fairly inexperienced with Proguard so was unclear which was the more appropriate solution